tracing-subscriber/src/fmt/fmt_layer.rs

```diff
@@ -12,6 +12,20 @@ use tracing_core::{
 
 /// A [`Layer`] that logs formatted representations of `tracing` events.
 ///
+/// ```rust
+/// use tracing_subscriber::{fmt, layer::Layer as _, registry::Registry, EnvFilter};
+///
+/// let default_filter = concat!(module_path!(), "=info");
+///
+/// let subscriber = EnvFilter::try_from_default_env()
+///     .or_else(|_| EnvFilter::try_new(default_filter))
+///     .unwrap()
```

Comment:
I don't really think any of this filter initialization stuff is directly relevant, so I'd prefer to leave it out to keep the example focused on _just_ the `fmt::Layer` type.

How about just
```rust
use tracing_subscriber{fmt, registry::Registry);
use tracing_subscriber::prelude::*;

let subscriber = fmt::Layer::default()
    .with_subscriber(Registry::default());
```

Reply:
okay, would it be okay to add an example to the `fmt` module itself that is more comprehensive rather than focusing on any single item.

Reply:
`tracing_subscriber::fmt::init();` :eyes:

Reply:
Im wondering how up to date the examples in the `fmt` module are, none of them seem to mention the Layer concept.